fluentd/lib/fluent-plugin-prometheus/in_prometheus_tail_monitor.rb

```diff
@@ -51,11 +51,11 @@ def start
         inode: get_gauge(
           :fluentd_tail_file_inode,
           'Current inode of file.'),
-        totalbytes_logged_in_disk: get_gauge(
-          :fluentd_totalbytes_logged_in_disk,
+        totalbytes_logged: get_gauge(
```

Comment:
Drop the bytes-logged metric, it isn't useful to generate it in fluentd. The same rotations are missed for both metrics, so we'll never see an accurate difference between logged and collected. We'll need to move that to Go.